pkgs/by-name/mu/multiqc/package.nix

```diff
@@ -0,0 +1,116 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  procps,
+  versionCheckHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "multiqc";
+  version = "1.26";
+
+  disabled = python3Packages.pythonOlder "3.8";
+
+  # Two data sources. One for the code, another for the test data
+  srcs = [
+    (fetchFromGitHub {
+      name = "multiqc";
+      owner = "MultiQC";
+      repo = "MultiQC";
+      rev = "v${version}";
+      hash = "sha256-MPAw6gG/3LzdskkDXOTDEM1NpG0sH9GvklYFQ1ZXWIs=";
+    })
+    (fetchFromGitHub {
+      owner = "MultiQC";
+      repo = "test-data";
+      rev = "67435083a8bfa228dca3dda7d835facef15fc2c7";
+      hash = "sha256-oYmPIJSy6dOKPcMr3B4foGoWcerA29x0XeGoU4dSYsA=";
+      name = "test-data";
+    })
+  ];
+
+  sourceRoot = "multiqc";
+
+  # Some tests run subprocess.run() with "ps" and multiqc""
+  postPatch = ''
+    substituteInPlace multiqc/plots/plotly/plot.py \
+      --replace-fail '"'ps'"' '"'${lib.getExe' procps "ps"}'"'
+    substituteInPlace tests/test_cmdline.py \
+      --replace-fail '["'multiqc'"' '["'$out/bin/multiqc'"'
```

Comment:
I think this can be removed and replaced with a preCheck where you update the PATH env. variable and add the directory $out/bin to it.

Reply:
I think what I'm doing here https://github.com/NixOS/nixpkgs/pull/370869 might help but it's not yet merged. Nevertheless, you can search in `nixpkgs` how the `PATH` is updated, especially in Python derivations.

Reply:
Indeed, thanks.

Reply:
Got your comment after pushing but it looks like `export PATH=$out/bin:$PATH` is often used in `preCheck`. I appended instead of prepending it but it seems to work.

Reply:
Yes, can you make sure to prepend it instead of appending it?

Reply:
Yes, it's corrected (and tests still passs).